modules/ocf_desktop/manifests/firefox.pp

```diff
@@ -0,0 +1,19 @@
+class ocf_desktop::firefox {
+  package {
+    'firefox':
+      ensure => present;
+    'iceweasel':
+      ensure => absent;
+  }
+
+  file {
+    # disable caching, history, blacklisting, and set homepage
+    '/etc/firefox/profile/prefs.js':
+      content => template('ocf_desktop/firefox/prefs.js.erb'),
+      require => Package['firefox'];
+    # start maximized by default
+    '/etc/firefox/profile/localstore.rdf':
+      source  => 'puppet:///modules/ocf_desktop/firefox/localstore.rdf',
+      require => Package['firefox']
+  }
+}
```

Comment:
I'd suggest moving this to `ocf::packages::firefox` since we include it outside of desktops (on the TV).

Then in extrapackages we can just `include ocf::packages::firefox` like we do for Chrome (though I think we don't include it in extrapackages?)